### ISSUE TITLE
Resume not supporting for Downloads if restart the download manager.

### ISSUE THREAD

**Issue by tuhinbhuiyan** [sjarvela/kloudspeaker#165]
Not sure if this resume downloads feature has been applied to kloudspeaker but I tested with IDM (Internet Download Manager). Where I have noticed once I start a normal download of a zip file. IDM starts to download the file in Resume Capability... Also even pause/resume do support as long as Download Manager session is open. But if I exit the Download Manager and restart it. Later if I try to resume the file. It just tell me that error "web site showing a web page instead of a file." and fails to resume downloading.  ![idm-klouderror](https://cloud.githubusercontent.com/assets/12027872/18842356/8a0c5dd2-8436-11e6-8725-a8a3e1e32608.png)

**Comment by sjarvela** [comment]
Is there a way to know what is it that it returns? Perhaps the session is not valid anymore, and it brings a login page etc, but I'd have to see the response.

Also setting debug mode can tell this by looking at the PHP error log after this error is shown.

**Comment by tuhinbhuiyan** [comment]
Yes, the problem is with the session is not valid. But I wonder why it is happening. Once closing down the IDM. Session of the link is not resumable.

Update: also checked in EgaleJet . It directly shows No Resume supported.
